v0/destinations/util.js

```diff
@@ -170,7 +170,32 @@ const getDataType = val => {
   return "string";
 };
 
-function safeTableName(provider, name = "") {
+const toSafeDBString = str => {
```

Comment:
There is a lot of warehouse specific code right now in this util. Might be a good time to move it out.


```
---warehouse/utils.js
  const v0 = require('v0/utils.js')
  const v1 = require('v1/utils.js')

  const getWarehouseUtil = function(schemaVersion) {
      if (schemaVersion == 'v0') {
         return v0;
      } if (schemaVersion == 'v1') {
         return v1;
      } else {
         // Default
         return v1;
      }
  }
```

```
---warehouse/v0/util.js
function safeTable() {
...
}
function safeColumn() {
...
}
function transformTableName() {
...
}
module.exports = {
safeTable,
safeColumn,
transformTableName
}
```
```
---warehouse/v1/util.js
function safeTable() {
...
}
function safeColumn() {
...
}
function transformTableName() {
...
}
module.exports = {
safeTable,
safeColumn,
transformTableName
}

 ```